src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -637,6 +639,267 @@ def _decode_inhibit_any_policy(backend, asn1_int):
     return x509.InhibitAnyPolicy(skip_certs)
 
 
+@utils.register_interface(x509.RevokedCertificate)
+class _RevokedCertificate(object):
+    def __init__(self, backend, x509_revoked):
+        self._backend = backend
+        self._x509_revoked = x509_revoked
+
+        self.__serial_number = None
```

Comment:
No problem. I just wanted to make sure they can not be accessed from outside of the class instead of just suggesting that they shouldn't be used.